### ISSUE TITLE
Some confusion regarding the use of the test dataset.

### ISSUE THREAD

**Issue by foolishcqx** [NimaFathi/AURA#2]
What an impressive job!

I’ve recently started working in this field and am trying to replicate your work. However, I’m a bit confused about the dataset used for evaluation in your paper. In your article, you mentioned:

“For evaluation, we utilize the held-out test set of the publicly available CheXpert dataset [11]. As AURA operates as an inference agent, leveraging off-the-shelf tools rather than being fine-tuned, this held-out set provides the sole relevant data for assessing its performance. To ensure a fair comparison and mitigate potential biases or distribution shifts, we adopt the identical data split used by PRISM [15].”

<img width="1075" height="365" alt="Image" src="https://github.com/user-attachments/assets/35f29e50-f505-4aa5-a0ea-b0620ae8310f" />

On the website, I found two download links: one for a 400+ GB file and another for a 5 GB "test image" dataset. Could you clarify which part of the dataset was used in your experiments? Specifically:

1. Did you use the 400 GB file and apply the 70-15-15 train-validation-test split, and then use the test portion for evaluation?
2. Or did you use the 5 GB "test image" dataset, applying the same 70-15-15 split, and then evaluate on the test portion?
3. Alternatively, did you directly use the test portion provided in the 5 GB "test image" file without any further splitting?

Your clarification would be really helpful for ensuring I’m using the correct dataset for reproduction.

Thank you!
